core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -605,6 +607,84 @@ public FileSystemMasterView getFileSystemMasterView() {
   }
 
   /**
+   * Checks the consistency of the subtree under the path.
+   *
+   * @param path the subtree root to check
+   * @param options the options to use for the checkConsistency command
+   * @return a list of paths in Alluxio which are not consistent with the under storage
+   * @throws FileDoesNotExistException if the path does not exist
+   * @throws InvalidPathException if the path is invalid
```

Comment:
I'd prefer to propagate this exception - it's not very clearly defined and I don't see why a reasonable application would want to identify it specifically